codesearch/index/merger.go

```diff
@@ -0,0 +1,62 @@
+package index
+
+import (
+	"io"
+
+	"github.com/cockroachdb/pebble"
+
+	"github.com/buildbuddy-io/buildbuddy/codesearch/posting"
+)
+
+const roaringMergerName = "roaring_merger"
```

Comment:
nit: can you add a comment about how this can never change without recreating the full index?

Reply:
good call, added